Bottom: abb7220ed155b6e9fdbc72dac81c97f21c587cfa
Top:    e71901eb2ef756ed3bd1440827c8ec4a2219914f
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-12 19:49:14 +0800

Refresh of autofs-5.0.5-wait-for-master-map-available-at-start.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index a9ee19e..7af6a76 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -184,7 +184,6 @@
 - fix out of order locking in readmap.
 - include ip address in debug logging.
 - mount using address for DNS round robin host names.
-<<<<<<< current
 - reset negative status on cache prune.
 - remove master_mutex_unlock() leftover.
 - fix sanity checks for brackets in server name.
@@ -196,9 +195,6 @@
 - add base64 password encode.
 - fix ipv6 name for lookup.
 - fix libtirpc ipv6 check.
-=======
-- wait for master map to be available at start.
->>>>>>> patched
 
 03/09/2009 autofs-5.0.5
 -----------------------
